painters/data_dirs_organizer.py

```diff
@@ -42,10 +42,12 @@ def _load_paths_labels_from_train_dir():
         paths.append(abspath_)
         labels.append(labels_lookup[name])
 
+    # LabelEncoder: Encode labels with value between 0 and n_classes-1
     return np.array(paths), LabelEncoder().fit_transform(labels)
 
 
 def _train_val_split_indices(labels):
+    # just 1 fold (so not really cross validation, but simply splits data into train and val)
```

Comment:
Nit: comments should be capitalized and end with a period.

Reply:
By capitalized, do you mean just the first letter (so the comment should be written as a compete sentence?)

Reply:
Yes, first letter capitalized with period.